waiter/src/waiter/util/utils.clj

```diff
@@ -482,13 +484,39 @@
 
 (defn map->base-64-string
   "Serializes data to a base 64 string along with encryption."
-  [data-map encryption-key]
-  (bytes->str (b64/encode (nippy/freeze data-map {:compressor nil :password encryption-key}))))
+  [data-map encryption-key & {:keys [url-safe] :or {url-safe false}}]
+  (let [encrypted-bytes (nippy/freeze data-map {:compressor nil :password encryption-key})]
+    (if url-safe
```

Comment:
It feels awkward to be using two different base 64 libraries based upon whether it's URL-safe.  Should we remove the dependency on `clojure.data.codec.base64`?

Reply:
Changed to use url-safe encoding. The saml tests have been rewritten to pass after the change.